lib/client/http-tracker.js

```diff
@@ -89,8 +89,14 @@ HTTPTracker.prototype._request = function (requestUrl, params, cb) {
   var self = this
   var u = requestUrl + (requestUrl.indexOf('?') === -1 ? '?' : '&') +
     common.querystringStringify(params)
+  var opts = {
+    url: u,
+    header: {
+      'user-agent': self.client._userAgent || ''
```

Comment:
I don't see where `self.client._userAgent` actually gets set. Did you forget to commit changes to `client.js`?